client/src/components/QA/QAAddQuestionModal.jsx

```diff
@@ -7,6 +7,7 @@ const QAAddQuestionModal = ({ productName, currentProductId }) => {
   const [question, setQuestion] = useState('');
   const [nickname, setNickname] = useState('');
   const [email, setEmail] = useState('');
+  const [name, setName] = useState(productName)
```

Comment:
Eventually, I think you could remove this, since you are using `productName` in the render anyway